crafting_combinator_xeraph/script/rc.lua

```diff
@@ -55,7 +55,51 @@ function _M.get_rc_slot_count()
 	return _rc_slot_count
 end
 
+local function add_compakt_circuits_integration()
+	if not remote.interfaces["compaktcircuit"] or not remote.interfaces["compaktcircuit"]["add_combinator"] then
```

Comment:
```suggestion
	if not script.active_mods["compaktcircuit"] then
```
I think it's better if this code errors out when `compaktcircuit` is active, but it hasn't defined this interface yet.
Otherwise, we might have hard-to-debug cases where everything looks like it loaded fine, but then placing the combinator in the processor doesn't work.

Reply:
And, well, I think check using `script.active_mods` is easier and more intuitive...